iree/test/e2e/regression/CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ iree_generated_trace_runner_test(
     "dylib"
     "vmvx"
   OPT_FLAGS
-    "--iree-flow-convert-linalg-matmul-to-mmt4d=M0=8 K0=4 N0=8"
+    "--iree-flow-convert-linalg-matmul-to-mmt4d=#target_info_for_each_variant#"
```

Comment:
Hmmm, I don't love this because it feels like something trying to be general, but actually very specific. I can't think of something better right now though, and it doesn't really cause issues, so I guess it's fine.

Reply:
i agree with the concern. i also think it's (just barely) good enough to allow us to proceed. there is one obvious alternative, per your comment, which is to make this explicitly specific instead of pretending to be general -- we could e.g. have a boolean `PASS_IREE_FLOW_CONVERT_LINALG_MATMUL_TO_MMT4D` option to iree_generated_trace_test_runner, etc.   On the other hand I thought that maybe what I'm doing here is actually not 100% specific, it's quite specific but the string "aarch64 dotprod [other_simd_features...]" is more generally useful in itself, it just happens to only be consumed by mmt4d at the moment.